fennel/_vendor/pydantic/env_settings.py

```diff
@@ -55,7 +55,7 @@ def _build_values(
         _env_nested_delimiter: Optional[str] = None,
         _secrets_dir: Optional[StrPath] = None,
     ) -> Dict[str, Any]:
-        # Configure built-in sources
+        # Configure built-in connectors
```

Comment:
looks like you made a replace all "sources" -> "connectors" :P 

These changes are in pydantic which are not relevant to this wider change